routing/routes.go

```diff
@@ -0,0 +1,24 @@
+package routing
+
+import (
+	"github.com/swift-sunshine/swscore/handlers"
+	"net/http"
+)
+
+type Route struct {
+	Name        string
+	Method      string
+	Pattern     string
+	HandlerFunc http.HandlerFunc
+}
+
+type Routes []Route
+
+var paths = Routes{
```

Comment:
This is a minor/non-blocking comment, but wouldn't it have been simpler to directly create routes using the chosen framework instead of having this intermediate `Routes` object?